src/firefly/js/templates/lightcurve/LCPanels.css

```diff
@@ -0,0 +1,26 @@
+.PanelResizableStyle {
+    width: 400px;
+    min-width: 400px;
+    height: 300px;
+    min-height: 300px;
+    overflow: auto;
+    background-color: #b3edff;
+    padding: 2px;
+    position: relative
+}
+
+.Header {
```

Comment:
Good point, noted.

On 9/13/16 1:24 PM, loitly wrote:
>
> In src/firefly/js/templates/lightcurve/LCPanels.css 
> <https://github.com/Caltech-IPAC/firefly/pull/172#discussion_r78636956>:
>
> > @@ -0,0 +1,26 @@
> > +.PanelResizableStyle {
> > +    width: 400px;
> > +    min-width: 400px;
> > +    height: 300px;
> > +    min-height: 300px;
> > +    overflow: auto;
> > +    background-color: #b3edff;
> > +    padding: 2px;
> > +    position: relative
> > +}
> > +
> > +.Header {
>
> CSS style name should follow BEM guildline: 
> http://getbem.com/introduction/
> Generic name like this may clash with others.
> We use '__' for blocks separator and '--' for modifiers.
> ie. LCPanels__Header[--if-needed]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Caltech-IPAC/firefly/pull/172/files/f4a60252e3ee7fdab9864a547846467f240344ba#r78636956>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ALaegy6YzFV3bcVoh_Pdbj3xl1tJtgv6ks5qpwaGgaJpZM4J5nMv>.
>